src/gobprepare/prepare_client.py

```diff
@@ -70,6 +74,7 @@ def connect(self):
         """
         self._src_connection, self._src_user = self._connect_application(self.source)
         self._dst_connection, self._dst_user = self._connect_application(self.destination)
+        self._dst_engine = self._create_engine(self.destination)
```

Comment:
Why use two different connection methods? Rather confusing. PostgresAPIImporter should use self._dst_connection as well.